### ISSUE TITLE
Android keyboard opens and closes if an input is inside the Siema

### ISSUE THREAD

**Issue by Willdta** [pawelgrzybek/siema#250]
If a user on an Android device decides to have an input inside one of their cards, the `resizeHandler` function overrides the user's ability to type anything, resulting in the keyboard opening and closing.

**Comment by Mativve** [comment]
I have the same problem, any ideas how to repair this?

**Comment by Mativve** [comment]
I have the same problem, any ideas how to repair this?
@pawelgrzybek

**Comment by Willdta** [comment]
@Mativve 

There is a built-in function `resizeHandler` that fired when either height/width changed (resize event).

I added the event listener back when:
- The width changed and the carousel was active

I removed the event listener when:
- The height changed, the carousel was active and mounted width was equal to current window width

However, my carousel was being opened by a button so it wasn't always shown which is why I had those checks. 

I hope this helps in some way!
